lib/api/model/model.dart

```diff
@@ -604,6 +613,51 @@ extension type const TopicName(String _value) {
   /// using [canonicalize].
   bool isSameAs(TopicName other) => canonicalize() == other.canonicalize();
 
+  /// Process this topic to match how it would appear on a message object from
+  /// the server.
+  ///
+  /// This assumes that the topic is constructed from a string without
+  /// leading/trailing whitespace.
```

Comment:
```suggestion
  /// This returns the [TopicName] the server would be predicted to include
  /// in a message object resulting from sending to this [TopicName]
  /// in a [sendMessage] request.
  ///
  /// This [TopicName] is required to have no leading or trailing whitespace.
```

The other part is needed too (I think it came from a comment @chrisbobbe made on the original PR): it says how this topic relates to the hypothetical scenario where the returned topic appears in a message object from the server.